src/components/TutorialCarouselDialog/index.js

```diff
@@ -68,14 +76,25 @@ export default class TutorialCarouselDialog extends Component {
   }
 
   render() {
-    const { classes } = this.props;
+    const { classes, langHandler, lang } = this.props;
     const { currentIndex, openTutorial } = this.state;
     const CurrentComponentName = this.components[currentIndex];
 
     return (
       <Dialog open={openTutorial} fullWidth maxWidth="md">
         <DialogContent className={cx(classes[`tutorialDialog${currentIndex}`], classes.tutorialDialog)}>
           <div className={classes.titleTopLine}></div>
+          <Select
+            className={classes.langBtn}
+            name="lang"
+            value={lang}
```

Comment:
can use `intl.locale` here instead of passing `lang` as prop

Reply:
done